hw/ip/pwm/data/pwm.hjson

```diff
@@ -0,0 +1,133 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "pwm",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  regwidth: "32",
+  param_list: [
+    { name: "NOutputs",
+      desc: "Number of PWM outputs",
+      type: "int",
+      default: "6",
+    }
+  ]
+  available_output_list: [
+    { name:  "pwm"
+      desc:  "pulse output"
+      width: "6"
+    }
+  ]
+  registers: [
```

Comment:
might be good to consider having a configuration lock.

Reply:
Sure.  This will be in the next commit.

Reply:
Added.  Check out the latest HJSON.